test/e2e/utils.go

```diff
@@ -0,0 +1,44 @@
+package e2e
+
+import (
+	"context"
+	"time"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+)
+
+func findInterface(name string, interfaces nmstatev1.State) bool {
+	//TODO: State is a interface{} we will have to convert it to json
+	//      and look for the interface name
+	return false
+}
+
+func waitInterface(namespace string, nodeName string, name string, mustExist bool) error {
+
+	f := framework.Global
```

Comment:
f is used only once in the following lines. could you just place it inline?

Reply:
Done